app/views/Main.js

```diff
@@ -53,7 +65,12 @@ export const Main = () => {
       AppState.removeEventListener('change', updateStateInfo);
       unsubscribe();
     };
-  }, [navigation, updateStateInfo]);
+  }, [
+    navigation,
+    updateStateInfo,
+    selectedAuthorities,
```

Comment:
can these just be split into two different useEffects? 
More broadly, I don't understand why we're doing the triggering background tasks depndent on changes to the redux state within the render cycle of a screen

Reply:
I created the redux middleware that triggers the intersection logic, which then starts the timer, so I was able to completely remove this logic from `Main.js` . Thanks for the advice on that